Source/CarthageKit/Xcode.swift

```diff
@@ -1032,6 +1064,9 @@ public func copyFramework(from: NSURL, to: NSURL) -> SignalProducer<NSURL, Carth
 	}
 }
 
+/// Alias of `copyFile` for backwards compatiblity.
+public let copyFramework = copyFile
```

Comment:
I included this because `copyFramework` is marked `public`. Let me know if you think this is unnecessary.